website/source/docs/vault-integration/index.html.md

```diff
@@ -0,0 +1,351 @@
+  under Nomad should have access to.
+
+*    `disallowed_policies` - Specifies the list of disallowed policies as a
+     comma-separated string. This list should contain all policies that jobs running
+     under Nomad should **not** have access to. The policy created above that
+     grants Nomad the ability to generate tokens from the token role should be
+     included in list of disallowed policies. This prevents tokens created by
+     Nomad from generating new tokens with different policies than those granted
+     by Nomad.
+
+     A regression occurred in Vault 0.6.4 when validating token creation using a
+     token role with `disallowed_policies` such that it is not usable with
+     Nomad. This will be remedied in 0.6.5 and does not effect earlier versions
+     of Vault.
+
+* `explicit_max_ttl` - Specifies the max TTL of a token. **Must be set to `0`** to
+  allow periodic tokens.
+
+* `name` - Specifies the name of the policy. We recommend using the name
+  `nomad-cluster`. If a different name is chosen, replace the token role in the
+  above policy.
+
+*   `orphan` - Specifies whether tokens created against this token role will be
+  orphaned and have no parents. Nomad does not enforce the value of this field
+  but understanding the implications of each value is important.
+
+    If set to false, all tokens will be revoked when the Vault token given to
+  Nomad expires. This makes it easy to revoke all tokens generated by Nomad but
+  forces all Nomad servers to use the same Vault token, even through upgrades of
+  Nomad servers. If the Vault token that was given to Nomad and used to generate
+  a tasks token expires, the token used by the task will also be revoked which
+  is not ideal.
+
+    When set to true, the tokens generated for tasks will not be revoked when
+  Nomad's token is revoked. However Nomad will still revoke tokens when the
+  allocation is no longer running, minimizing the lifetime of any task's token.
+  With orphaned enabled, each Nomad server may also use a unique Vault token,
+  making bootstrapping and upgrading simpler. As such, **setting `orphan = true`
+  is the recommended setting**.
+
+* `period` - Specifies the length the TTL is extended by each renewal in
+  seconds. It is suggested to set this value on the order of magnitude of 3 days
+  (259200 seconds) to avoid a large renewal request rate to Vault. **Must be set
+  to a positive value**.
+
+* `renewable` - Specifies whether created tokens are renewable. **Must be set to
+  `true`**. This allows Nomad to renew tokens for tasks.
+
+The above [`nomad-cluster` token role](/data/vault/nomad-cluster-role.json) is
+available for download. Below is an example of writing this role to Vault:
+
+```
+# Download the token role
+$ curl https://nomadproject.io/data/vault/nomad-cluster-role.json -O -s -L
+
+# Create the token role with Vault
+$ vault write /auth/token/roles/nomad-cluster @nomad-cluster-role.json
+```
+
+
+#### Example Configuration
```

Comment:
This section is repetitive and seems unnecessary to me.

Reply:
New guide located at /guides/operations/vault-integration/index.html